static/css/style.css

```diff
@@ -286,101 +195,28 @@ article {
   height: 100%;
   background-image: url('/img/cover.png');
   opacity: 0.33;
+  filter: blur(3px);
```

Comment:
remove this. The built in blur looks super bad. Just remove it. I prefer it without it.